Realm/ObjectStore/impl/apple/external_commit_helper.cpp

```diff
@@ -32,6 +32,87 @@
 using namespace realm;
 using namespace realm::_impl;
 
+#if TARGET_OS_TV
```

Comment:
On the async queries branch I've mostly unified the two versions so that nearly all of the code is shared, but that depends on the earlier restructuring.